### ISSUE TITLE
Hosts and Getting started

### ISSUE THREAD

**Issue by OilyGoats** [Maxx53/SCMBot#388]
Hi, so I recently downloaded the SCM Bot and have been running it off a 24/7 VPS on Amazon AWS. I am currently in Australia and have set up the VPS in Oregon, USA as it is closest to valve HQ. It has been running for over 3 days now and i have seen an ak47 vulcan sell for under a $1 and a flip knife fade seller for 3 cents and the bot was watching both. Also, what is the recommended maximum number of items to watch at once. Also, what is the difference between the items page and the newly listed page? With the default hosts i am getting pretty bad ping considering the location of the server and was wondering how I can change it as others have to get better connection. Any help would be greatly appreciated. Cheers, Josh. 
![scmbot](https://cloud.githubusercontent.com/assets/13101245/8406885/93c13bc0-1ea4-11e5-99b0-daf270d1dd95.jpg)

**Comment by Maxx53** [comment]
i think hosts feature is pretty useless.
just use search and you'll find all answers

**Comment by Alex1os** [comment]
Well, look at the thread <i opened a while ago  https://github.com/Maxx53/SCMBot/issues/367
the better ping times with hosts helped me get my dream knife more than once btw

Lg Alex

**Comment by OilyGoats** [comment]
Hey, thanks for the response. I have seen your thread a while ago but when i changed the ips i get a html error.

**Comment by Alex1os** [comment]
yes that is normal, u have to find ips that work with the bot, most ips give html error

**Comment by OilyGoats** [comment]
Hi again, could you please share the ips that you used for your hosts? I am running a VPS out of Frankfurt, Germany. Thanks, Josh.
